pkg/api/common_test.go

```diff
@@ -167,7 +160,6 @@ func (sc *scenarioContext) exec() {
 
 type scenarioFunc func(c *scenarioContext)
 type handlerFunc func(c *models.ReqContext) response.Response
-type handlerFuncCtx func(ctx context.Context, c *models.ReqContext) response.Response
```

Comment:
Time to remove `handlerFuncCtx`, since Context is available in `c.Req.Context()` anyway.